packages/main/src/plugin/container-registry.ts

```diff
@@ -550,7 +558,9 @@ export class ContainerProviderRegistry {
             }),
           );
         } catch (error) {
-          console.log('error in engine', provider.name, error);
+          if (this.notify) {
+            console.log('error in engine', provider.name, error);
+          }
```

Comment:
nit: replace all if console.log by a method notifyConsole('message') that will do the if